idea_town/frontend/static-src/app/views/landing_page.js

```diff
@@ -0,0 +1,37 @@
+import app from 'ampersand-app';
+import AmpersandView from 'ampersand-view';
+import mustache from 'mustache';
+
+// TODO replace with an api endpoint that exposes idea town addon info
+const addonInfo = {
+  name: 'Idea Town',
+  url: '//localhost/codez/github/mozilla-idea-town-addon/@idea-town-addon-0.0.1.xpi'
```

Comment:
hmm, would it be worthwhile to just have a script to dl the addon into a place to be served with our static files? If you think this will be replaced soon then not a big deal, we can just create an issue.